src/microbe_stage/editor/MicrobeEditorCheatMenu.cs

```diff
@@ -5,13 +5,20 @@ public class MicrobeEditorCheatMenu : CheatMenu
     [Export]
     public NodePath? InfiniteMpPath;
 
+    [Export]
+    public NodePath? RevealEntirePatchMapPath;
```

Comment:
Only the first nodepath should be nullable. All other ones should be `= null!` initialized.

The dispose method should be rewritten as:
```
if(InfiniteMpPath != null){
  InfiniteMpPath.Dispose();
  RevealEntirePatchMap.Dispose();
}
```